crates/graphql/src/schema/objects/nft.rs

```diff
@@ -3,7 +3,7 @@ use indexer_core::assets::{AssetIdentifier, ImageSize};
 use objects::{bid_receipt::BidReceipt, listing_receipt::ListingReceipt};
 use reqwest::Url;
 
-use super::prelude::*;
+use super::{prelude::*, purchase_receipt::PurchaseReceipt};
```

Comment:
```suggestion
use super::prelude::*;
```
import `objects::purchase_receipt::PurchaseReceipt`, cargo fmt should merge it with the existing imports from `objects` on line 1